### DISCUSSION TITLE
An Unresolved reference 'insert' in iOS land on the calls to String.insert( char, at: position)

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#190] [Q&A] [answered]
The call to insert a character inside a String is basic Swift...

Why would this not behave?  What could I be doing wrong now?

https://developer.apple.com/documentation/swift/string/insert(_:at:)-1uqvy

![Screenshot 2024-08-14 at 9 39 09 PM](https://github.com/user-attachments/assets/8f857941-867e-4f17-accd-a30e3047b6c9)

**Comment by marcprux** [comment, marked as answer]
As mentioned in the SkipLib reference at https://skip.tools/docs/modules/skip-lib/#swift-standard-library-support, Kotlin strings are not mutable, and so we cannot support functions like `insert`. The workaround is to do something like:

```
greet = atSign + greet
```
